pkg/skaffold/parser/config_test.go

```diff
@@ -402,6 +403,30 @@ requires:
 				createCfg("cfg01", "image01", ".", nil),
 			},
 		},
+		{
+			description:              "recursively applied profiles",
+			profiles:                 []string{"pf0"},
+			applyProfilesRecursively: true,
+			documents: []document{
+				{path: "skaffold.yaml", configs: []mockCfg{{name: "cfg00", requiresStanza: `
+requires:
+  - path: doc1
+    configs: [cfg10]
+`}, {name: "cfg01", requiresStanza: ""}}},
+				{path: "doc1/skaffold.yaml", configs: []mockCfg{{name: "cfg10", requiresStanza: `
+requires:
+  - path: ../doc2
+    configs: [cfg21]
+`}, {name: "cfg11", requiresStanza: ""}}},
+				{path: "doc2/skaffold.yaml", configs: []mockCfg{{name: "cfg20", requiresStanza: ""}, {name: "cfg21", requiresStanza: ""}}},
```

Comment:
Can you put this last one on a separate line?
```suggestion
				{path: "doc2/skaffold.yaml", configs: []mockCfg{{name: "cfg20", requiresStanza: ""},
				{name: "cfg21", requiresStanza: ""}}},
```